pages/keplr/notification-page.js

```diff
@@ -1,5 +1,9 @@
 const approveButton = `button`;
+const copyAddress = 'Copy Address';
+const copyWalletAddressSelector = 'div.sc-dkzDqf div.sc-hKMtZM.sc-kDDrLX.cyoEAq.dkJSBQ'
 
```

Comment:
At the moment, using a selector to copy cosmos address. This is temporary as I have to add an implementation for selecting agoric chain in our `importWallet` flow. Then I'll update the selector based on that.